testing/btest/scripts/base/frameworks/logging/delay/errors.zeek

```diff
@@ -0,0 +1,52 @@
+# @TEST-DOC: Test some error cases
+
+# @TEST-EXEC: zeek  -b -r $TRACES/http/get.trace %INPUT
+# @TEST-EXEC: TEST_DIFF_CANONIFIER='$SCRIPTS/diff-remove-abspath | sed -r "s/0x[0-9a-z]+/0x<...>/g"' btest-diff .stderr
+
+redef enum Log::ID += { LOG };
+
+event zeek_init()
+	{
+	# Not within a log write
+	Log::delay(LOG, []);
```

Comment:
This results in `no active write context available`, right? I'm wondering whether we need to make this more explicitly about not being called from a global policy hook. Might also be worth a test case calling this from a filter-level policy hook. (I haven't absorbed all the tests, sorry if that's elsewhere.)

Reply:
> This results in no active write context available, right?

Ah, yes, not the best message for a user.

> Might also be worth a test case calling this from a filter-level policy hook. 

I think there is one, I need to look, too :sweat_smile: